extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeSubscriberTest.java

```diff
@@ -0,0 +1,1178 @@
+package org.schabi.newpipe.extractor.services.youtube;
+
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.schabi.newpipe.DownloaderTestImpl;
+import org.schabi.newpipe.extractor.NewPipe;
+import org.schabi.newpipe.extractor.exceptions.ExtractionException;
+import org.schabi.newpipe.extractor.localization.Localization;
+import org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeChannelExtractor;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+import static org.schabi.newpipe.extractor.ServiceList.YouTube;
+import static org.schabi.newpipe.extractor.utils.Utils.removeNumber;
+
+/**
+ * Test for {@link }
+ * and specifically YoutubeChannelExtractor.getSubscriberCount()
+ * in all the languages supported by YouTube.
+ * Takes a long time because we need to test make 146 requests to YouTube
+ * DON'T RUN ON MOBILE DATA
```

Comment:
I don't know what you want me to do here.
And yeah I'll write more why I let commented

Reply:
I didn't want you to do anything here. Just wanted to say that is better to not run the tests every time, but also keep the file in case we encounter a problem.

Reply:
Should I comment the imports too? Or it makes no change?